hello-vulkan/app/build.gradle

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (C) 2022 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
+
+android {
+    compileSdk 33
+    ndkVersion '25.1.8937393'
+    
+    defaultConfig {
+        shaders {
+            glslcArgs.addAll(['-c'])
+        }
+        applicationId 'com.android.hellovk'
+        minSdkVersion 30
+        targetSdkVersion 33
```

Comment:
nit: do you want to give it a try to drop the "Version" from minSdk and targetSdk? a little more consistent, but really minor.